internal/topo/context/default.go

```diff
@@ -39,12 +39,12 @@ const (
 )
 
 type DefaultContext struct {
-	ruleId           string
-	opId             string
-	instanceId       int
-	ctx              context.Context
-	err              error
-	enableRuleTracer bool
+	ruleId         string
+	opId           string
+	instanceId     int
+	ctx            context.Context
+	err            error
+	isTraceEnabled *isRuleTraceEnabledWrapper
```

Comment:
Why need to use a wrapper? Just want to add the lock. Is it possible to use atomic.bool to keep it simple？